packaging/rpm/microshift.spec

```diff
@@ -221,6 +221,24 @@ The microshift-gateway-api-release-info package provides release information fil
 release. These files contain the list of container image references used by Gateway API
 and can be used to embed those images into osbuilder blueprints.
 
+%package ai-model-serving
+Summary: AI Model Serving for MicroShift
+ExclusiveArch: x86_64
+Requires: microshift = %{version}
+
+%description ai-model-serving
+The microshift-ai-model-serving package provides manifests for the RHOAI based kserve and serving runtimes.
+
+%package ai-model-serving-release-info
+Summary: Release information for AI Model Serving for MicroShift
+BuildArch: noarch
+Requires: microshift = %{version}
+
+%description ai-model-serving-release-info
+The microshift-ai-model-serving-release-info package provides release information files for this
+release. These files contain the list of container image references used by Model Serving
+and can be used to embed those images into osbuilder blueprints.
```

Comment:
As we now also have bootc, we should probably changing the wording by removing "osbuilder"?

Reply:
Whatever we decide to do, please update all the relevant places.